app/src/main/res/raw/rooms_info.json

```diff
@@ -297,111 +297,111 @@
       }
     },
     {
-      "roomCode":842,
+      "roomCode":"842",
       "floorCode":"H-8",
       "centerCoordinates":{
         "longitude":-73.5789343714714,
         "latitude":45.49737026836251
       }
     },
     {
-      "roomCode":"elevator",
+      "roomCode":"elevators-2",
```

Comment:
Are we not distinguishing staff elevators from elevators accessible to everyone?

Reply:
Fixed to have two different types, "elevators-#" and "staff elevators-#"